MetaRock.rm

```diff
@@ -0,0 +1,148 @@
+between the metaverse and the everyday individual.
+
+
+
+### Why is this project is good for the Darwinia ecosystem:
+
+MetaRock will build a market place on Darwinian and Crab network allowing Users to mint and sell NFTs natively in the Darwinia Ecosystem.
+Metarock will also allow Darwinia Network's existing community to start trading and selling NFT art and resources on the Darwinia and Crab Blockchains
+bringing utility to Ring and Crab.
+
+
+### How you will integrate this project into Darwinia:
+in the early stages Metarock will use the rmrk1.0 shortly followed by rmrk2.0 and ERC 721. Metarock is already minting NTFs on the Crab and Pangolin Blockchain 
+with Plans to build a community backed Liquidity pool to help build the communities local Dex liquidity. Once our Beta is released we will enable the community 
+and Darwinia to start minting and selling NFTs 
+
+
+### Team
+
+- Josiah Kotzur ( Darwinia Ambassador / Darwinia Development team )
+- James ( Software engineer )
+- Ela ( UI/UX Designer )
+- Pili ( Graphic Designer )
+
+
+
+### Legal Structure
+Currently Metarock has no defined legal structure other then a Dapp created by  Kotzur Marine Industries  Pty Ltd  trading as Blockchain Integration  ABN 41 126 551 379 
+
+
+
+### Team Code Repos
+
+- https://github.com/Doinglifewell (Josiah Kotzur)
+- https://github.com/TopCodeBeast/MetaRock_RMRK.git (James)
+
+
+
+### Development Roadmap
+
+MetaRock Beta: ( current )
+
+- Platform testing and UI/UX optimisation 
+- Basic NFT scan tool
+- AR/VR mobile capabilities 
+- 3D assist display capabilities 
+- Buy, sell, destroy and send functions
+
+
+Bellow is Meta Rocks long term draft roadmap which has been displayed to indicate our direction over the next 1-3 years and is not to be used to measure the 
+completion of the Darwinia Collaboration grant. 
+
+MetaRock 1.0:  
+- MetaClone Tool ( cloning tool used to clone Parent NFTs as they are sold )
+- Build on Numoriouse Blockchains and allow the Community to start selling items in their currency and generate their own income within that ecosystem.
+- Plug into metaverses allowing users to sell items in games and designers to sell Skins and ingame items.
+- Commence building DeFi Liquidity pools allowing buyers to buy in any currency and sellers to receive any currency as payment.
+
+Metarock 2.0: 
+
+- Liquidity pools 
+- Currency exchange allows sellers to receive funds in a specific currency and buyers to use a specific currency to make purchases. 
+- Referral based NFT sales smart contracts
+- Royalties 
+- NFT based access into Advance Minting Platform. 
+
+
+MetaRock 3.0:
+
+- Income tracking system / Profit and Loss report
+- Onchian Profiles 
+- Consist of smart contract based NFTs and goods ordering system. 
+- Insurance 
+- Privacy functions 
+
+
+
+
+Please provide the following:
+
+
+## Colabriation Funding  
+- 6,000USD issued in RING once Marketplace has released and MetaRock.app Beta version is released ( Mid January 2022 )
```

Comment:
As RING value fluctuates, it would be better to specify a concrete RING number equates to 6000 USD. A reasonable exchange rate might be the one of pull request day, or average rate during following days.

Reply:
I'm would like to make a change to this section and make it USDT payable in ERC20 tokens in 2 stages. 

Stage 1: $2,000USDT payable on acceptance of the grant and collaboration partnership.
Stage 2: $4,000USDT payable once Metarock has released its Beta version and passes the acceptance test.

If you are fine with this please approve it here and I will update Metarock.rm file